remme/clients/pub_key.py

```diff
@@ -76,20 +80,21 @@ def process_csr(self, certificate_request, key, not_valid_before=None, not_valid
             not_valid_after
         ).sign(key, hashes.SHA256(), default_backend())
 
-    def store_certificate(self, certificate_raw, signature_rem, signature_crt, cert_signer_public_key=None):
-        payload = self.get_new_certificate_payload(certificate_raw,
-                                                   signature_rem,
-                                                   signature_crt,
-                                                   cert_signer_public_key)
+    def store_pub_key(self, public_key, entity_hash, entity_hash_signature, valid_from, valid_to):
```

Comment:
this should accept certificate, as it was before. 
but instead of storing certificate_raw data, 
this method should extract public key from cert and pass it to handler.

Reply:
Extraction going in the function `get_crt_export_bin_sig_rem_sig`

Reply:
Oh. I see